### ISSUE TITLE
XeofilteR for other species

### ISSUE THREAD

**Issue by aliibarry** [PeeperLab/XenofilteR#29]
Hi, nice package! Have you trialled this against other species with well annotated genomes (eg. rat)?

**Comment by RoelKluin** [comment]
Yes, other species should also work. It is also possible to align against a custom fasta with HPV sequences and human for instance. Only if the alternative reference also includes human sequence, then reads aligning there will be filtered.
